### PR TITLE
♻ Api endpoint rename

### PR DESCRIPTION
**Pull request by SentientHousePlant**
# Api endpoint rename

1 extra character to rule them all....

...this PR adds a "s" so that all the endpoints have a plural resource name...

...it changes /application/ to /applications/...

### CODE REVIEW

File: api/namespace/applications/routes.py [hunk 1]
```diff
@@ -1,6 +1,11 @@
 from database.store import APPLICATIONS
 from api.namespace.applications.applications_ns import applications_ns
```
Comment (harryyo) [inline api/namespace/applications/routes.py:2]: Is this line now duplicated below?

File: api/namespace/applications/routes.py [hunk 2]
```diff
@@ -51,3 +56,70 @@ def get(self):
             "Access-Control-Allow-Credentials": True
         }
         return APPLICATIONS.search_applications(args), 200, response_headers
+
```
Comment (harryyo) [inline api/namespace/applications/routes.py:59]: Should there be a second line here (call me flake8)

File: api/namespace/applications/routes.py [hunk 3]
```diff
@@ -51,3 +56,70 @@ def get(self):
...
+        new_status = args["new_status"]
+
+        status_update = APPLICATIONS.update_status(
+            application_id, question_name, new_status
+        )
+
+        if status_update:
+            return 200
+        else:
+            return 404
```
Comment (harryyo) [inline api/namespace/applications/routes.py:125]: And a newline here?

**Review by harryyo — APPROVED** [review]
I only (ever) have PEP8 queries...

**Review by ramsharma-prog — APPROVED** [review]
Great job

**Pull request merged**